clustered/client/src/main/resources/ehcache-clustered-ext.xsd

```diff
@@ -97,6 +95,51 @@
     </xs:restriction>
   </xs:simpleType>
 
+  <xs:complexType name="cluster-connection-spec">
+    <xs:sequence maxOccurs="1" minOccurs="1">
```

Comment:
These are the default min and max values no need to specify them.

Reply:
Indeed. Done.